src/main/webapp/app/shared/layouts/navbar/navbar.component.html

```diff
@@ -37,7 +37,7 @@
                 routerLinkActive="active"
                 [routerLinkActiveOptions]="{ exact: true }"
             >
-                <a class="guided-tour-course-admin nav-link" routerLink="course-management" (click)="collapseNavbar()" id="course-admin-menu">
+                <a *ngIf="!examModeActive()" class="guided-tour-course-admin nav-link" routerLink="course-management" (click)="collapseNavbar()" id="course-admin-menu">
```

Comment:
Just out of interest. Why do you put it on the a element and no on the li element?

Reply:
because Angular doesn´t like two *-attributes on the same element 😄